payments_service/solitude/__init__.py

```diff
@@ -52,6 +52,7 @@ def _api_request(self, method, *args, **kw):
             log.warn('{api}: solitude returned 400: {details}'
```

Comment:
totally unrelated but I realize this message is wrong. It should be ``solitude returned {status_code}: ...``